### PR TITLE
this commit is of oop, while and for loop and private class program with description

### PR DESCRIPTION
**Pull request by 7nishant**

### CODE REVIEW

File: nishant_work/oop.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+class Bird:
+
+    def __init__(self):
+        print("Bird is ready")
+
+class Penguin(Bird):
+
+    def __init__(self):
+        super().__init__()
```
Comment (Patibandha) [inline nishant_work/oop.py]: What is super().__init__() does?

File: nishant_work/oop.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+#What is OOP? What are the OOP concept for python? explain with programming.
+# Object-oriented Programming, or OOP for short, is a programming paradigm which provides a means of structuring
+# programs so that properties and behaviors are bundled into individual objects. Python is a great programming
+# language that supports OOP. You will use it to define a class with attributes and methods, which you will then call.
+# Python offers a number of benefits compared to other programming languages like Java, C++ or R. It's a dynamic
+# language, with high-level data types. This means that development happens much faster than with Java or C++. It does
+# not require the programmer to declare types of variables and arguments. This also makes Python easier to understand
+# and learn for beginners, its code being more readable and intuitive.
+
+#Inheritance :-A process of using details from a new class without modifying existing class.Inheritance is a way of
```
Comment (Patibandha) [inline nishant_work/oop.py]: This is not the correct definition, it does what you explain over here but it does more then that!

File: nishant_work/oop.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+class Parrot:
+
+    def fly(self):
+        print("Parrot can fly")
+
+    def swim(self):
+        print("Parrot can't swim")
+
+
+class fish:
```
Comment (Patibandha) [inline nishant_work/oop.py]: class suppose to be `Fish` not `fish`

File: nishant_work/oop.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+    def fly(self):
```
Comment (Patibandha) [inline nishant_work/oop.py]: ditto as above

File: nishant_work/private.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+#How to write private class in python? give program for example.
```
Comment (Patibandha) [inline nishant_work/private.py]: This entire answer is for how to create private and protected variable. but the question is how to create private class in python not variable

File: nishant_work/while_for.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
+#What is different of while and for loop? which one is fast & why ? Write program for example.
+
```
Comment (Patibandha) [inline nishant_work/while_for.py]: All answer is correct, very good! but you need to change the program over here, you need to show with bigger loop which on is faster using timestamp

File: nishant_work/oop.py (hunk 1 above)
Comment (7nishant) [reply nishant_work/oop.py]: when we need to initialize the parent class with in the subclass then this function make the process easier...

**Pull request closed without merging**